.readthedocs.yml

```diff
@@ -1,23 +1,11 @@
 ---
-# .readthedocs.yml
-# Read the Docs configuration file
-# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
-
-# Required
 version: 2
 
-# Build documentation in the docs/ directory with Sphinx
-sphinx:
-  builder: html
-  configuration: docs/conf.py
-
-# Optionally build your docs in additional formats such as PDF and ePub
-formats: []
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.9"
 
-# Optionally set the version of Python and requirements required to build your
-# docs
 python:
-  # FIXME(willkg): 2021-04-29: ReadTheDocs doesn't support 3.9
-  version: 3.8
   install:
-    - requirements: docs/requirements.txt
+    - requirements: requirements.txt
```

Comment:
This updates to the current v2 format. Further, ReadTheDocs supports Python 3.9, so we can go back to a less complicated way of building them in the local dev environment.